thirdparty/grpc/remove-custom-commands-for-unused-dependencies.patch

```diff
@@ -0,0 +1,425 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index fc4b6a40a5..ae494f1464 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -489,7 +489,7 @@ if(WIN32)
+   set(_gRPC_ALLTARGETS_LIBRARIES ${_gRPC_ALLTARGETS_LIBRARIES} ws2_32 crypt32)
+   set(_gRPC_STATIC_WIN32 STATIC)
+ endif()
+-  
++
+ if(BUILD_SHARED_LIBS AND WIN32)
+ 
+ # Currently for shared lib on Windows (i.e. a DLL) certain bits of source code
```

Comment:
This section of the patch looks like it's just a whitespace difference. Is it needed?

Reply:
No it's not needed, my editor automatically removes line ending whitespaces so it probably added this when I created the diff.

Reply:
Removed in https://github.com/apache/nifi-minifi-cpp/pull/1897/commits/18fc86102c9cce88d4189e2187a7653d629c5cf7

Reply:
Readded the whitespace change and added a note to be aware of the reverse patch dry run issue in https://github.com/apache/nifi-minifi-cpp/pull/1897/commits/116696f1b51ee689bb2e36aa6187489216d39d39